packages/iov-ethereum/src/testconfig.spec.ts

```diff
@@ -20,14 +22,27 @@ export interface EthereumNetworkConfig {
   readonly wsUrl: string;
   readonly chainId: ChainId;
   readonly minHeight: number;
-  /** one account with fixed state */
-  readonly accountState: {
-    readonly pubkey: PublicKeyBundle;
-    readonly address: Address;
-    /** expected balance for the `address` */
-    readonly expectedBalance: Amount;
-    /** expected nonce for the `address` */
-    readonly expectedNonce: Nonce;
+  readonly accountStates: {
+    /** An account with ETH and ERC20 balance */
+    readonly full: {
```

Comment:
This is a strange name. Suggest `ethAndErc20`?

Reply:
Hmm the account should be used in all kind of tests (e.g. nonce tests) and I don't want to restrict the naming to ETH+ERC20. Thus I came up with `full`. Maybe `default` is better?